File: DSA/RATNA_KHANNA_590013141/DAY_46_QUESTION_3.c
#include <stdio.h>
#define MAX 100
int main() {
    int Q1[MAX], Q2[MAX], merged[MAX];
    int n1, n2, i, j, k = 0;
    printf("Enter number of elements in Queue 1: ");
    scanf("%d", &n1);
    printf("Enter %d elements for Queue 1:\n", n1);
    for (i = 0; i < n1; i++) {
        scanf("%d", &Q1[i]);
        merged[k++] = Q1[i]; 
    }
    printf("Enter number of elements in Queue 2: ");
    scanf("%d", &n2);
    printf("Enter %d elements for Queue 2:\n", n2);
    for (j = 0; j < n2; j++) {
        scanf("%d", &Q2[j]);
        merged[k++] = Q2[j];  
    }

    // Output merged queue
    printf("Merged Queue: ");
    for (i = 0; i < k; i++) {
        printf("%d ", merged[i]);
    }
    printf("\n");

    return 0;
}

File: DSA/NiharikaChand_590017697/Day46Question3.java
import java.util.*;
class Day46Question3
{
    public static Queue<Integer> merge(Queue<Integer> q1, Queue<Integer> q2) 
    {
        Queue<Integer> result = new LinkedList<>();

        for (int num : q1) 
        {
            result.offer(num);
        }
        for (int num : q2) 
        {
            result.offer(num);
        }
        return result;
    }

    public static void main(String args[]) 
    {
        Scanner sc = new Scanner(System.in);
        Queue<Integer> q1 = new LinkedList<>();
        Queue<Integer> q2 = new LinkedList<>();

        System.out.print("Enter number of elements in Queue 1: ");
        int n1 = sc.nextInt();
        System.out.println("Enter integers for Queue 1:");
        for (int i = 0; i < n1; i++) 
        {
            q1.offer(sc.nextInt());
        }
        System.out.print("Enter number of elements in Queue 2: ");
        int n2 = sc.nextInt();
        System.out.println("Enter integers for Queue 2:");
        for (int i = 0; i < n2; i++) 
        {
            q2.offer(sc.nextInt());
        }
        Queue<Integer> mergedQueue = merge(q1, q2);
        System.out.println("Merged Queue: " + mergedQueue);
    }
}

